content/sources/workflow.adoc

```diff
@@ -24,21 +24,20 @@ image::git-organization.png[Git Organization, 600, align=center, link={imagesdir
 This repository is the developers repository. The main development branch is `develop` branch. Depending the contribution kind (one shot contribution or feature contribution), the developer can use a dedicated `feature` branch if needed.
 The most of development activity will be done in this repository by a lot of contributors :
 
-* `maintainers` (formerly known as `sl3`) : To develop fix. These developments are done in dedicated `fix` branches (more details below).
-* `all development teams` : To do one shot contribution and handle feature branch life cycle.
+* `maintainers` (formerly known `Maintenance FT`) : To develop fix. These developments are done in dedicated `fix` branches (more details below).
+* `all other feature teams` : To do one shot contribution and handle feature branch life cycle.
 
 In this repository, `develop` branch and `feature` branch will be under CI.
 
 === Blessed repository
 
 In this repository you can find `stable` branches and release tags.
-Only some people are able to write on this repository :
+Only some people use to write on this repository :
```

Comment:
People can still push on this repo to fix bugs.

Reply:
done